infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -33,6 +33,64 @@ nightly_builds = [
 
 # Built on push to specific tag.
 versioned_builds = [
+  # Remove libtpu from PyPI builds
+  {
+    git_tag         = "v2.4.0-rc1"
+    package_version = "2.4.0-rc1"
+    pytorch_git_rev = "v2.4.0-rc1"
+    accelerator     = "tpu"
+    python_version = "3.8"
```

Comment:
From line 5 - 30, I found Python 3.8 is removed from nightly, let's drop 3.8 in this release as well. cc @will-cromar